runtime/util/jbcmap.c

```diff
@@ -247,7 +247,7 @@ const unsigned char sunJavaInstructionSizeTable[] = {
 0x00 /* JBunimplemented = 222 = 0xDE */,
 0x00 /* JBunimplemented = 223 = 0xDF */,
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
-0x01 /* JBdefaultvalue = 224 = 0xE0 */ ,
+0x03 /* JBdefaultvalue = 224 = 0xE0 */ ,
```

Comment:
Referring to `page 36` in http://cr.openjdk.java.net/%7Efparain/L-world/L-World-JVMS-4b.pdf, `defaultvalue
 = 203 (0xcb)`. You need to update `jbcmap.c`.

From - `0x03 /* JBdefaultvalue = 224 = 0xE0 */ ,`

To - `0x03 /* JBdefaultvalue = 203 = 0xCB */ ,`

Reply:
As with withfield:

It looks like there are currently some internal bytecodes using 203 and 204 at the moment (iloadw and lloadw according bcnames.c). I think we decided to go with a hack in ClassFileOracle.cpp:1392 to remporarily remap 204 to 226 and 203 to 224 until a final LWVT spec is released.

Reply:
there are several other places where the opcodes will need to be remapped (`bcnames.h`, `bcnames.c`, `cfr.h`, `bcsizes`, etc.). Should I leave the same comments in all of those locations?